### PR TITLE
added more comments information about prometheus metrics

### PR DESCRIPTION
**Pull request by talaviss**
**What this PR does / why we need it**:
This PR better explains what is prometheus package and what capabilities it adds to the Kubernetes solution
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
This are comments and it's not fixing an issue
**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by talaviss** [comment]
I signed it!

**Comment by talaviss** [comment]
I signed it!

File: pkg/version/prometheus/prometheus.go [hunk 1]
```diff
@@ -15,7 +15,8 @@ limitations under the License.
 */
 
 // Package prometheus registers Kubernetes version information as a
-// prometheus metric.
+// prometheus metrics which is a visualization solution to power your metrics
```
Comment (lavalamp) [inline pkg/version/prometheus/prometheus.go]: I don't feel super strongly, but I think just the link is probably enough information.

**Comment by lavalamp** [comment]
Sorry for delayed response, I was OOO most of last month.

**Comment by mbohlool** [comment]
@talaviss please assign back to me when you address lavalamp's comment.

**Comment by talaviss** [comment]
@mbohlool .  I don't understand from lavalamp comment which actions should I take in order to approve this pull request.

**Comment by mbohlool** [comment]
@talaviss The approval message is from k8s-bot not lavalamp. I was talking about @lavalamp's comment on your code. Are you saying you don't understand that comment?

**Comment by grodrigues3** [comment]
Closing due to inactivity

**Pull request closed without merging**